.github/workflows/workflow.yml

```diff
@@ -86,6 +86,17 @@ jobs:
           shopt -s globstar
           openfisca test tests/**/*.py
 
+  test-path-length:
```

Comment:
If I understand correctly and this path length is for Windows, shouldn't this job be part of “test on Windows”? 🙂

Reply:
I've also added it to test-on-windows just in case: https://github.com/openfisca/openfisca-france/actions/runs/3295747292/jobs/5434844978#step:4:7